### ISSUE TITLE
after reload electron throws error when resizing or moving window

### ISSUE THREAD

**Issue by aavelyn** [Quramy/electron-connect#52]
Hi! I'm using the same code as your example (gulp) to reload electron.
The reloading works great but I'm getting the following error as soon as I move or resize the window

```
Error: Attempting to call a function in a renderer window that has been closed or released. Function provided here: client.js:82:19.
    at BrowserWindow.callIntoRenderer (/Users/hanche/Desktop/bitbucket/Finished-Setup/node_modules/electron-prebuilt/dist/Electron.app/Contents/Resources/electron.asar/browser/rpc-server.js:189:19)
    at emitOne (events.js:101:20)
    at BrowserWindow.emit (events.js:188:7)
```

I'm using electron inside angular 2 and in the index.html I'm currently using an relative path to access electron connect (../node_modules/electron-connect), not sure if this causes it? Restarting doesn't throw the issue.

Cheers

**Comment by ssreekanth** [comment]
Thank you @aavelyn for reporting the issue. I have fixed it with [bf7d255](https://github.com/Quramy/electron-connect/commit/bf7d255063b0e7d13aa945ed6689f6c459eb6d71). Let me know if you still notice any issue.

**Comment by ssreekanth** [comment]
Closing this out.
